File: 2020S2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    long long n, m, k;
    cin >> n >> m >> k;

    vector<int> row(n + 1, 0);
    vector<int> col(m + 1, 0);

    for (int i = 0; i < k; i++) {
        char t;
        long long x;
        cin >> t >> x;

        if (t == 'R') row[x] ^= 1;
        else col[x] ^= 1;
    }

    long long R = 0, C = 0;
    for (int x : row) R += x;
    for (int x : col) C += x;

    long long gold = R * (m - C) + C * (n - R);
    cout << gold << "\n";
}

File: 2020S2.py
n = int(input())  # rows
m = int(input())  # columns
k = int(input())  # operations

row = [0] * (n + 1)
col = [0] * (m + 1)

for _ in range(k):
    t, x = input().split()
    x = int(x)
    if t == 'R':
        row[x] ^= 1
    else:
        col[x] ^= 1

R = sum(row)
C = sum(col)

gold = R * (m - C) + C * (n - R)
print(gold)
